src/components/buttons/Button.vue

```diff
@@ -27,8 +27,7 @@ withDefaults(
   }
 );
 
-const emit = defineEmits<{ (e: 'click', event: MouseEvent): void }>();
-
+// try comment
```

Comment:
Any reason why we removed the click and replaced it with a comment? 😄

Reply:
haha, sorry, I'll remove the comment.
that line is no longer needed, the click was added because of vue 2, vue 3 handles everything through useAttrs()